### ISSUE TITLE
"Back to homepage" button needs more padding

### ISSUE THREAD

**Issue by shreyachatterjee00** [civiform/civiform#9391]
![image](https://github.com/user-attachments/assets/6886bbf1-19e1-4eae-8c3a-b8aeb040e3a6)

**Comment by dkatzz** [comment]
@sijiayam How much padding would you recommend here?

**Comment by EmiliaPaz** [comment]
I'm a new Google.org fellow working on CiviForm. Trying to get familiar by looking at good first bugs. Got a couple questions:
- How can I enable the North Star feature on a local build?
- What is the URL that shows the "Back to homepage" button with the padding bug? Or reproduction steps?

Would appreciate any pointers to get started :)

**Comment by rockycodes** [comment]
@EmiliaPaz To enable North Star, login as a CiviForm admin using the devtools, then click on the Gear icon in the upper right hand corner to go to the admin settings panel. Under the "Experimental" section, select "true" for the `NORTH_STAR_APPLICANT_UI` flag.

I'm pretty sure this is the "upsell" view you see as an applicant after completing an application that prompts you to create an account or sign-in. But looking at it now it seems like it might not be an issue anymore? @shreyachatterjee00 what do you think?

<img width="732" alt="Image" src="https://github.com/user-attachments/assets/8f453c83-deaf-416e-997a-4e343856178d" />

**Comment by shreyachatterjee00** [comment]
@rockycodes good point, looks like the design of that page was updated a bit so this is no longer an issue. I'll close this and @EmiliaPaz I can find you another issue to work on!

**Comment by sijiayam** [comment]
@shreyachatterjee00 @rockycodes is there any instance when the "other programs you might be interested in" wouldn't show?

**Comment by rockycodes** [comment]
Great callout @sijiayam when there are no other programs, you don't see that section and the padding is weird:

<img width="787" alt="Image" src="https://github.com/user-attachments/assets/cca9a339-a8a3-4c46-893e-5e8cf1966306" />

**Comment by shreyachatterjee00** [comment]
OH yes that makes sense! Thanks for calling that out @sijiayam. 

@EmiliaPaz Sorry for all the confusion! Looks like there is still a fix to be implemented here. You can recreate this by turning ON the `NORTH_STAR_APPLICANT_UI` flag, as well as make sure the `SUGGEST_PROGRAMS_ON_APPLICATION_CONFIRMATION_PAGE` flag is OFF.

**Comment by shreyachatterjee00** [comment]
FYI, kind of related - it looks like even if you turn the `programs on confirmation screen` flag off, the programs still show in NS. Filed a bug for that here: https://github.com/civiform/civiform/issues/9631

**Comment by sijiayam** [comment]
We have a space variable called `page` that should be used here. Here is the relevant mock:

[Confirmation page - no related programs](https://www.figma.com/design/RmhVJCLgAKWGENJLWWAtNS/CiviForm%3A-Components%2C-Styles?node-id=2933-13459&m=dev)
[Spacing guide](https://www.figma.com/design/RmhVJCLgAKWGENJLWWAtNS/CiviForm%3A-Components%2C-Styles?node-id=1614-7047&m=dev)

cc @EmiliaPaz

**Comment by EmiliaPaz** [comment]
Thanks for the info @sijiayam. I can't see the variable on the confirmation page, but looking at the spacing guide `page` uses 4 USWDS. PR #9649 adds `<div class="page-spacing"></div>` after the button, which adds u-height(4). I think this is the same as 4 USWDS, but please let me know if it's not
